lib/models/widget.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function create_widget( $name, $id, $description ) {
+
+  	register_sidebar(array(
+  		'name' => __( $name ),
+  		'id' => $id,
+  		'description' => __( $description ),
+  		'before_widget' => '<div class="widget">',
+  		'after_widget' => '</div>',
+  		'before_title' => '<h3>',
+  		'after_title' => '</h3>'
+  	));
+
+  }
+
+  create_widget( 'Page Sidebar', 'page', 'Displays on the side of pages with a sidebar' );
```

Comment:
Been using a namespacing pattern, for example: https://github.com/SandersForPresident/SitesForBernie/blob/develop/lib/init.php

Want to add `namespace SandersForPresident\Wordpress\Widgets;` to the top of this file?